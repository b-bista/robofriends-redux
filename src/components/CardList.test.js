import { shallow, mount, render} from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('tests that cardList is rendered once', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'John Snow',
            username: 'John John',
            email: 'john@gmail.com'
        }
    ]

    expect.assertions(1);
    expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
})