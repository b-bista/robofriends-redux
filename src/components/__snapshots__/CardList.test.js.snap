// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests that cardList is rendered once 1`] = `ShallowWrapper {}`;
